### DISCUSSION TITLE
Unable to install on NetBeans 19 RC

### DISCUSSION THREAD

**Discussion by manikantannaren** [apache/netbeans#6246] [General]
When attempting install on NB19 release candidate, the plugin installer complains of unmet dependencies. 
I have OpenJDK 20 on Osx Ventura.
<img width="531" alt="image" src="https://github.com/apache/netbeans/assets/5435047/2ea6945c-be6a-4a30-91bf-2aef15378f11">

**Comment by neilcsmith-net** [comment]
Please open an issue for this, and provide as much info on OS and JDK architecture, vendor and version as you can, as well as the relevant section of the log when this happens. Thanks!
